sources/packages/backend/libs/utilities/src/address-utils.ts

```diff
@@ -0,0 +1,17 @@
+import { AddressInfo } from "@sims/sims-db";
+
+// 'selectedCountry' in the student contact info will have the value 'other',
+// when 'Other'(i.e country other than canada) is selected.
+export const OTHER_COUNTRY = "other";
+// 'selectedCountry' in the student contact info will have the value 'canada',
+// when 'Canada' is selected.
+export const COUNTRY_CANADA = "canada";
```

Comment:
They are both essentially negations of each other. Do we need both of them? From what I could understand, just one is enough to resolve all the logic conditions. Please let me know if I am missing something.

Reply:
These constants are not new and were just moved from one lib to another.
They can potentially become one but I would not see additional refactoring around this as part of this PR.